lightyear/src/client/prediction/rollback.rs

```diff
@@ -113,6 +125,7 @@ pub(crate) fn check_rollback<C: SyncComponent>(
     // We use Option<> because the predicted component could have been removed while it still exists in Confirmed
     confirmed_query: Query<(Entity, Option<&C>, Ref<Confirmed>)>,
     rollback: Res<Rollback>,
+    mut evw_rollback: EventWriter<RollbackEvent>,
```

Comment:
This would prevent the `check_rollback` systems for multiple components from running in parallel, so I'm not sure it's something we want to have currently unless we redesign those systems to have only one system for all predicted components

Reply:
Hmm, good point. How about if `RollbackEvent` is generic on the component type? It'll be a bit more work for users to aggregate but it won't prevent paralellism